src/data/GameTypeMappings.tsx

```diff
@@ -19,9 +19,61 @@ import {
   faInbox,
   faServer,
 } from '@fortawesome/free-solid-svg-icons';
+import { HandlerGameType } from 'bindings/HandlerGameType';
+import { GameType } from 'bindings/InstanceInfo';
+
+type InstanceTab = {
+  title: string;
```

Comment:
can you try to pull 0.4.3 and handle merge? I'm worried about merge conflict with Kevin's editable titles and stuff. He changed instance tabs iirc